release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -1234,6 +1242,13 @@ As of {product-title} 4.14, the `--cloud` parameter for the `oc adm release extr
 
 For more information, see xref:../release_notes/ocp-4-14-release-notes.adoc#ocp-4-14-install-update-cco-manual-enhancements[Simplified installation and update experience for clusters with manually maintained cloud credentials].
 
+[id="ocp-4-14-dep-docker-locations"]
+==== Using Docker configuration locations for registry credentials is now deprecated
+
+As of {product-title} 4.14, using Docker configuration file locations, such as `~/.docker/config.json`, to store registry credentials for `oc` commands is deprecated. Instead, use Podman configuration file locations, such as `${XDG_RUNTIME_DIR}/containers/auth.json`.
```

Comment:
```suggestion
As of {product-title} 4.14, using Docker configuration file locations, such as `~/.docker/config.json`, to store registry credentials for `oc` commands is deprecated. Instead, use Podman configuration file locations, such as `${XDG_RUNTIME_DIR}/containers/auth.json` or `${HOME}/.config/containers/auth.json.
```

This might be good to mention as the first one is not persistent and the second one is.